lib/utils/__tests__/atRuleRawsAfterNameIndex.test.mjs

```diff
@@ -0,0 +1,28 @@
+import atRuleRawsAfterNameIndex from '../atRuleRawsAfterNameIndex.mjs';
+import postcss from 'postcss';
+
+describe('atRuleRawsAfterNameIndex', () => {
+	it('had a "media" condition', () => {
+		expect(atRuleRawsAfterNameIndex(atRule('@media (color) {}'))).toBe(6);
+	});
+
+	it('has a "supports" condition', () => {
+		expect(atRuleRawsAfterNameIndex(atRule('@supports (color) {}'))).toBe(9);
+	});
+
+	it('has a space before the @', () => {
+		expect(atRuleRawsAfterNameIndex(atRule(' @media (color) {}'))).toBe(6);
+	});
+
+	it('has a newline before the param', () => {
```

Comment:
nit

If `@media` is not "param", can you change the description?

Reply:
Good catch, thank you